### DISCUSSION TITLE
can not join the telegram group

### DISCUSSION THREAD

**Discussion by Schueni1** [Hypfer/Valetudo#2162] [Q&A / Support] [answered]
### Are you already using Valetudo?

No, but planning to do it in the near future

### Your questions

When I try to join the telegram group, I always get a message, that I have been blocked by an admin.

**Comment by Hypfer** [comment, marked as answer]
If you're being instantly kicked, that's the automod.
Because telegram's moderation tools are rather subpar, the bot enforces people to have a screen name with at least two real characters OR a username. Without that, you simply can't search for a specific user + their messages, which makes moderating pretty much impossible.

The automod bot also disallows a bunch of undesirable nonsense usernames like "empty" (using some unicode hacks) names or zalgo text.

Unfortunately, bots can only talk to people after those people first talked with the bot, meaning that there can't be any feedback on that on telegram, making this a bit confusing.


Additionally, everyone that does join successfully currently gets muted for 3 minutes. This is to encourage people to think first instead of just blabbing away. That's why it's 3 minutes. 5 minutes would've been a bit excessive, but no minutes makes for too little thinking.

It also turned out to help keep some bots away, which start thinking that they've been found out, and immediately abort their mission, apparently.
